syntax/import.go

```diff
@@ -11,11 +11,12 @@ import (
 	"strings"
 	"sync"
 
+	"github.com/anz-bank/sysl/pkg/mod"
```

Comment:
Nit: Is there a way to avoid introducing this dependency? e.g. move the module logic into a separate package like https://github.com/anz-bank/pkg?

Reply:
good point. will do it in a separate pr.